pkg/manager/internal/controllers/organizationroletemplate_controller.go

```diff
@@ -96,3 +156,71 @@ func (r *OrganizationRoleTemplateReconciler) handleDeletion(ctx context.Context,
 
 	return nil
 }
+
+func (r *OrganizationRoleTemplateReconciler) reconcileRBACForOrganization(ctx context.Context, organizationRoleTemplate *corev1alpha1.OrganizationRoleTemplate, organization *corev1alpha1.Organization) error {
+	// Reconcile Role.
+	role := &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      organizationRoleTemplate.Name,
+			Namespace: organization.Status.Namespace.Name,
+		},
+		Rules: organizationRoleTemplate.Spec.Rules,
+	}
+	if err := r.reconcileRole(ctx, role, organizationRoleTemplate); err != nil {
+		return err
+	}
+
+	// Reconcile RoleBindings.
+	if organizationRoleTemplate.HasBindTo(corev1alpha1.BindToOwners) {
```

Comment:
```suggestion
	if organizationRoleTemplate.HasBinding(corev1alpha1.OwnersBinding) {
```

Reply:
I have updated it to:
`if organizationRoleTemplate.HasBinding(corev1alpha1.BindToOwners)`
Is that ok with you?

Reply:
good enough